#data
n_frames: 4
data_freq: 50 #freq of the dataset (when collected)
target_freq: 1 #freq of the dataset (when processed)
hidden_size: 64

#training
lr: 0.001
epochs: 1000000
batch_size: 128
load_dir: /home/alfredo/canopies/code/CanopiesSimulatorROS/workspace/src/imitation_learning/data
save_dir: ./params #/home/adriano/Desktop/canopies/code/CanopiesSimulatorROS/workspace/src/imitation_learning/src/agent_params
param_tag: encode
task: grasp
device: cuda
#